server/models/Appointment.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const Appointment require('./Appointment');
+const Pet require('./Pet');
+const Service require('./Service');
+
+const appointmentSchema = new Schema({
+  date: {
+    type: Integer,
+    required: true,
+  },
+  time: {
+    type: Integer,
```

Comment:
How are we saying time as in integer? Are we using 1400 instead of 2pm?

Reply:
In our last project, we use simple numbers to represent the hours (8, 9, 10, 11, 12, 13, 14, 15, 16...) which makes it easier than using timestamps and or strings. Also, the same explanation of the Scalar Type for Graphql would apply in this case as well.

Reply:
A timestamp won't be the correct way to represent time. Are we going to allow the user to filter times? If we use ints we are going to need to format it on the UI.  Mongo can represent DateTimes objects as strings too. We should make this kind of design decisions as a team.

Reply:
I will explain the logic later, but it is not a timestamp. You won't need to do any formating.